### PR TITLE
Mijn 669 responsive my updates and my chapters

### PR DESCRIPTION
**Pull request by BennievanderWel**

### CODE REVIEW

File: src/components/MainHeader/MainHeader.module.scss [hunk 1]
```diff
@@ -90,6 +91,7 @@
 @include respond-to(medium-screen) {
   .topBar {
     height: $responsive-header-topbar-height;
+    max-width: 100%;
```
Comment (timvanoostrom) [inline src/components/MainHeader/MainHeader.module.scss]: This is unnecessary as 94rem is the max width for all sizes

File: src/pages/Dashboard/Dashboard.module.scss [hunk 2]
```diff
@@ -2,27 +2,31 @@
 
 .Dashboard {
   @include select(PageContentMainBody) {
-    display: flex;
-    flex-direction: row;
-    flex-wrap: wrap;
+    width: 100%;
```
Comment (timvanoostrom) [inline src/pages/Dashboard/Dashboard.module.scss]: For block level elements that are not absolutely positioned a width of 100% is redundant as block level elements are 100% width by default.

File: src/components/MainHeader/MainHeader.module.scss (hunk 1 above)
Comment (BennievanderWel) [reply src/components/MainHeader/MainHeader.module.scss]: Cool

File: src/pages/Dashboard/Dashboard.module.scss (hunk 2 above)
Comment (BennievanderWel) [reply src/pages/Dashboard/Dashboard.module.scss]: OK

**Pull request merged**